plastron/commands/__init__.py

```diff
@@ -3,3 +3,11 @@ def __init__(self, config=None):
         if config is None:
             config = {}
         self.config = config
+
+    # Enable repository config dictionary to be overridden by the command before
+    # actually creating the respository.
+    #
+    # The default implemented of this method simply returns the provided
+    # repo_config dictionary without change
+    def override_repo_config(self, repo_config, args={}):
```

Comment:
@dsteelma-umd Rather than name this `override_repo_config`, which feels like it is exposing the OO-mechanics unnecessarily, it could just be named `repo_config` or `get_repo_config`.